_posts/2021-05-21-js-waku.md

```diff
@@ -0,0 +1,171 @@
+2021-05-21-js-waku.md---
+layout: post
+name:  "Presenting JS-Waku: Waku v2 in the browser"
+title:  "Presenting JS-Waku: Waku v2 in the browser"
+date:   2021-05-21 12:00:00 +0800
+author: franck
+published: true
+permalink: /2021-05-21-js-waku
+categories: platform
+summary: "We are summarizing the Waku v2 status, introducing JS-Waku, what we achieved so far and what is next."
+discuss: 
```

Comment:
Make sure this is added later

Reply:
how is it used? I could not find the link (when published on the web page) on previous posts

Reply:
What I meant is: Why does this field needs to be filled?
I can now see that a text at the bottom of the post is automatically added to point to discuss. will sort it.